sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -442,6 +515,65 @@ def hpat_pandas_series_iloc_impl(self):
 @sdc_overload_attribute(SeriesType, 'loc')
 def hpat_pandas_series_loc(self):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.loc
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series_loc/series_loc_single_result.py
+       :language: python
+       :lines: 27-
+       :caption: With a scalar integer. Returns single value.
+       :name: ex_series_loc
+
+    .. command-output:: python ./series_loc/series_loc_single_result.py
+       :cwd: ../../../examples
+
+    .. literalinclude:: ../../../examples/series_loc/series_loc_multiple_result.py
+       :language: python
+       :lines: 27-
+       :caption: With a scalar integer. Returns multiple value.
+       :name: ex_series_loc
+
+    .. command-output:: python ./series_loc/series_loc_multiple_result.py
+       :cwd: ../../../examples
+
+    .. literalinclude:: ../../../examples/series_loc/series_loc_slice.py
+       :language: python
+       :lines: 27-
+       :caption: With a slice object. Returns multiple value.
+       :name: ex_series_loc
+
+    .. command-output:: python ./series_loc/series_loc_slice.py
+       :cwd: ../../../examples
+
+    .. note::
```

Comment:
I think there should be limitation block.
```
    Limitations
    -----------
```
Or something like that.
@PokhodenkoSA ?